src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -581,6 +583,43 @@ public async Task<ActionResult<Instance>> Delete(int instanceOwnerPartyId, Guid
             return Ok(updatedInstance);
         }
 
+        /// <summary>
+        /// Updates the presentation fields of an instance
+        /// </summary>
+        [Authorize(Policy = AuthzConstants.POLICY_INSTANCE_WRITE)]
+        [HttpPut("{instanceOwnerPartyId:int}/{instanceGuid:guid}/presentationFields")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [HttpPut]
+        public async Task<Instance> UpdatePresentationFields(
+            [FromRoute] int instanceOwnerPartyId,
+            [FromRoute] Guid instanceGuid,
+            [FromBody] Dictionary<string, string> presentationFields)
```

Comment:
Wouldn't that mean having another level before being able to access the presentation fields? 
rather that accessing instance.PresentationFields there would be yet another level? If that is the case I'm not sure I agree that we should make this change.

Reply:
The new concrete class can be a dictionary. I think it is enough to make it extend Dictionary<string, string>. You'll need to try.

Reply:
I've now taken a look at the resulting OpenAPI specification and it doesn't look bad. However, what I'm concerned with is what it will look like once the specification is imported somewhere. In API Management we already have some really brilliantly named types:

` Instances-instanceOwnerPartyId-instanceGuid-EventsPost201ApplicationJsonResponse`
`App-org-app-Get200ApplicationJsonResponse`

And I want to avoid more of them by defining concrete types for requests and responses on public endpoints.

Reply:
created a new class. Waiting to push nugets and reference in project 'til pr is approved

Reply:
We need to use the new class here as input to the method as well. Otherwise, clients that use the specification might invent type names, like the examples above. Changing the type of the `PresentationFields` property on `Instance` was less important, but a good change for consistency.

Reply:
Yes. I can't do that untill after the Storage.Interface nuget is pushed.. If you think the model is okay I can pushe that packange and update the references in the code.